pennylane/ops/__init__.py

```diff
@@ -81,7 +81,11 @@ def _heisenberg_rep(p):
     def diagonalizing_gates(self):
         return []
 
+    @staticmethod
+    def identity_op(*params):
```

Comment:
Could we have not used the `Identity._matrix` class method instead of this def?

Would be nice to add a short docstring here.

Reply:
I did use that initially, but then I opted to write a static method because I didn't like calling the private attribute from a different file.

Reply:
Added doc string here